### DISCUSSION TITLE
允许工单重新执行

### DISCUSSION THREAD

**Discussion by LeoQuote** [hhyo/Archery#1723] [Ideas]
有时候工单审批完成了, 但是因为各种的原因执行失败了, 此时需要重新执行就要重新走一遍流程, 实际上只是需要重跑一遍, 不需要重新审批才对

如果能支持重新执行, 应该可以省去一些麻烦

**Comment by hhyo** [comment]
执行过程是非幂等的，并且也不是逐条保存执行结果，允许重复执行风险较大，不过用户需求确实存在，可以设计一下

**Reply by LeoQuote** [reply]
可能可以单独给"重复"执行开一个权限
